Automattic/wp-calypso
# A4A: add finish signup screen

Related to https://github.com/Automattic/automattic-for-agencies-dev/issues/357

## Proposed Changes

This PR wraps up the work on the Streamline signup flow. After successful login/signup we redirect user to `signup/finish` where we finish registration.

@jkguidaven could you please verify if loading Hubspot script here is done properly?


## Testing Instructions



- Using current branch (preferrable through the local build) verify different scenarios of navigating to `/signup` endpoint and flows

## Pre-merge Checklist
